Tests/MapboxCoreNavigationTests/LocationTests.swift

```diff
@@ -1,6 +1,7 @@
 import XCTest
 import CoreLocation
 @testable import MapboxCoreNavigation
+#if !SWIFT_PACKAGE
 @testable import TestHelper
```

Comment:
I conditionally uncompiled any tests that depend on TestHelper. Unfortunately, that’s just about all the interesting tests in MapboxCoreNavigationTests. MapboxCoreNavigationTests had been depending on TestHelper, which depends on MapboxNavigation. We’ll eventually need to break out the MapboxNavigation-dependent parts of TestHelper into a separate target.

Reply:
#2792